src/sagas/tokens.js

```diff
@@ -133,7 +137,7 @@ function* fetchTokenHistory(action) {
     }
 
     const response = yield call(wallet.getTxHistory.bind(wallet), { token_id: tokenId });
-    const data = response.map((txHistory) => mapTokenHistory(txHistory, tokenId));
+    const data = response.map(mapToTxHistory(tokenId));
```

Comment:
Why did you change this here but in the actions is different?

<img width="733" alt="image" src="https://github.com/HathorNetwork/hathor-wallet-mobile/assets/3298774/284314a4-ebac-49ec-b2e1-66979724b187">